synapsemonitor/monitor.py

```diff
@@ -113,6 +113,46 @@ def _find_modified_entities_file(syn: Synapse, syn_id: str, days: int = 1) -> li
     return []
 
 
+def _traverse(
+    syn: Synapse, synid_root: str, include_types: list = ["folder", "file", "project"]
```

Comment:
Use typing.List: https://stackoverflow.com/questions/39458193/using-list-tuple-etc-from-typing-vs-directly-referring-type-as-list-tuple-etc

Reply:
Changed to typing.List